phpBB/styles/prosilver/template/icons/iconify.html

```diff
@@ -0,0 +1,4 @@
+{% spaceless %}
+	<i class="iconify o-icon{{ CLASSES ? ' ' ~ CLASSES }}"{% if S_HIDDEN %}{% if TITLE %} title="{{ lang(TITLE) }}"{% endif %} aria-hidden="true"{% endif %} data-icon="{{ ICON }}" data-inline="true"{{ ATTRIBUTES }}></i>
```

Comment:
change the `iconify` class to `iconify o-icon-svg` since this will return an svg icon.